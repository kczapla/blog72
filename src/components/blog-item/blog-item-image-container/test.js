import React from 'react'
import renderer from 'react-test-renderer'

import BlogItemImageContainer from './index'

describe('blog item image container', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<BlogItemImageContainer post={ { image: 'tmp' } }/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
