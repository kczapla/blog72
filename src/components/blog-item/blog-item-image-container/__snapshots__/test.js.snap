// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`blog item image container renders correctly 1`] = `
<div
  className="sc-bdfBwQ iTesnw"
>
  <img
    className="sc-gsTCUz rQlxY"
    src="tmp"
  />
</div>
`;
